recipes/scikit-fdiff/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "scikit-fdiff" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scikit-fdiff-{{ version }}.tar.gz
+  sha256: 05d39e607c553cbc3b3ec328b84a6f2665291cd095ef94ae8479819ad4fbe3ec
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - numpy
+    - numba
+    - scipy
+    - sympy <1.6
+    - streamz
+    - more-itertools
+    - xarray
+    - dask-core
+    - pyyaml
+    - joblib
+    - path
+    - tqdm
+    - cloudpickle
+    - attrs
+    - matplotlib-base
+    - holoviews
+    - boltons
+    - appdirs
+    - python-forge
+    - fuzzywuzzy
+    - cachetools
+    - cached-property
+    - loguru
+
+test:
+  imports:
+    - skfdiff
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://gitlab.com/celliern/scikit-fdiff/
+  summary: Automatic finite difference discretization for 1D PDE with fast temporal solvers.
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - thealanjason
+    - conda-forge/python
```

Comment:
You shouldn't list `conda-forge/python` as a co-maintainer.

Reply:
Fixed